src/bindings/python/fluxacct/accounting/user_subcommands.py

```diff
@@ -149,7 +149,8 @@ def update_default_bank(conn, cur, username):
     if len(result) > 0:
         # update user rows to have a new default bank (the next earliest user/bank created)
         new_default_bank = result[0][0]
-        edit_user(conn, username, default_bank=new_default_bank)
+        # pylint: disable=no-value-for-parameter
+        edit_user(conn, username=username, default_bank=new_default_bank)
```

Comment:
Does the decorator confuse pylint about what the arguments are? That's unfortunate. 

Does the decorator force to provide `username` here as a keyword argument or did you make that change for pylint?

Reply:
Yeah, unfortunately pylint reports an error here unless explicitly suppressed. I went ahead and provided `username` as a keyword argument to hopefully make the function call a little clearer, but I don't think it's entirely necessary to pass it as a keyword argument. Do you have a preference?